Tools/Anonymization/README.md

```diff
@@ -0,0 +1,33 @@
+# Anonymization App
+
+This is a text anonymization app for users to anonymize different parts of a text that are deemed to be private and confidential (e.g. student id, handphone number, name of individual, etc.). In addition, this app also provides the option to remove single line and block comments.
+
+## Usage
+
+1. Ensure that all the files to be anonymized are placed under the `Test` folder. 
+
+2. Then, enter the below command:
+
+<pre> python main.py --add [ANONYMIZER_TYPES] --rm_single_comments [BOOLEAN_OPTION] --rm_block_comments [BOOLEAN_OPTION] </pre>
+
+- --add ANONYMIZER_TYPES refers to the anonymizer types that you would like the application to detect and then replace such occurances. If you want to include all the anonymizer types that this application can detect, simply enter `--add all`. Else, if you want to selectively add anonymizer types, refer to the below section.
+
+- --rm_single_comments indicates whether do you want the app to remove all single line comments from the given input files (to be placed under `Test` folder as written above). If you want the app to remove all single line comments, enter `--rm_single_comments yes`. Otherwise, enter `--rm_single_comments no`.
```

Comment:
Do you use a lexer (e.g., antlr4 or the more recent one) for this?  Some language's comments may be more difficult to detect properly.  (e.g., HTML's really ugly `<!-- -->`

Reply:
This only works for C/C++/Java/Python files. I have made appropriate changes to point out this in the documentation and code.